tests/unit/twiml/test_voice_response.py

```diff
@@ -624,3 +641,14 @@ def text_mixed(self):
             self.strip(r),
             '<?xml version="1.0" encoding="UTF-8"?><Response>before<Say>Content</Say>after</Response>'
         )
+
+    def test_add_child(self):
+        with VoiceResponse() as r:
+            with r.add_child('alexa', omnipresent='true') as alexa:
+                alexa.add_child('purchase', 'Kindle')
+
+        assert_equal(
+            self.strip(r),
+            '<?xml version="1.0" encoding="UTF-8"?><Response><alexa '
```

Comment:
it's not critical, but for readability I would put the line breaks in between tags